src/integ_test/java/games/strategy/engine/lobby/server/login/LobbyLoginValidatorIntegrationTest.java

```diff
@@ -118,7 +118,7 @@ public void testAnonymousLoginBadName() {
       // this is probably a duplicate insertion error, we can ignore that as it only means we already added the bad
       // word previously
     }
-    assertEquals(LobbyLoginValidator.THATS_NOT_A_NICE_NAME,
+    assertEquals(LobbyLoginValidator.ErrorMessages.THATS_NOT_A_NICE_NAME,
```

Comment:
Just a note for the next breaking release.
We should probably not send the exact error message, but return a string like "invalid-name-error" which is then mapped to a human readable error message on the client side to enable localization